vscode/src/player/Player.ts

```diff
@@ -69,6 +79,7 @@ export default class Player {
     this.audioPlayer.play(timeToStart);
     this.timer.run(timeToStart);
     this.isPlaying = true;
+    this.updateContext('isPlaying', this.isPlaying);
```

Comment:
Consider moving `isPlaying` and `inCodioSession` to a const

Reply:
Will make that move.